docs/developer/app-store/apps/crm.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: CRM
+sidebar_position: 3
+---
+
+# CRM
+
+## Overview
+
+_CRM_ is a Saleor App that allows providing customer data to external services. Here are the currently available providers:
+
+- [Mailchimp](http://mailchimp.com/)
+- [Twilio Segment](https://segment.com/) [_coming soon_]
+- [RudderStack](https://www.rudderstack.com/) [_coming soon_]
+
+There are two main features CRM App provides
+
+- Initial index of existing customers stored in Saleor and sending them to the provider contact list
+- Creating a new contact in the provider, when a new Customer is created or updated in Saleor
+
+:::caution
+
+To successfully configure the CRM App, you must:
+
+- Have a Mailchimp account
+- Have `MANAGE_APPS` permission
+
+:::
+
+## Installation
+
+The App can be installed from Saleor App Store. To install the App, you must have `MANAGE_USERS` and `MANAGE_APPS` permissions.
+
+## Configuration
+
+The App is configured per provider. You can configure more than one provider - the App will work with at least one provider set up
+
+### Mailchimp configuration
+
+#### Authorization
+
+App uses Mailchimp OAuth flow, meaning it will ask you to log in with Mailchimp and connect the App with your Saleor account. You can always disconnect the App from Mailchimp dashboard.
+
+After successful authorization, App will store a token used to connect with a Mailchimp API.
+
+You can disconnect your Mailchimp account and "forget" the token from App/Saleor from the App configuration screen. The token will be removed and you will have to connect with Mailchimp again to continue using it.
+
+To permanently disconnect the App, you have to visit the Mailchimp dashboard.
+
+#### Customer creation webhook
+
+The App allows sending a new or updated customers to Mailchimp via a dedicated Saleor webhook. By default, this feature is disabled.
+When you enable it, you have to select what Mailchimp [audience list](https://mailchimp.com/developer/marketing/api/lists/) contact should be synchronized to (likely you have only one).
+
+The App will set a customer in Mailchimp when customer is created (`CUSTOMER_CREATED` event) and updated (`CCUSTOMER_UPDATED` event).
+
+#### Customers bulk sync
+
+If you install the App and already have previously created customers, you would likely want to perform initial index of customers and send the to the provider.
+
+You can do that from the App.
```

Comment:
This sentence feels like something is missing here. I suggest providing some more context about where and how the user can do it.